docs/sources/mobile-app/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Grafana OnCall mobile app
+aliases:
+  - /docs/oncall/latest/mobile-app/
+keywords:
+  - Mobile App
+  - oncall
+  - notification
+  - push notification
+weight: 1200
+---
+
+# Grafana OnCall Mobile App
+
+>**Note**: This application is currently in beta
+
+The Grafana OnCall mobile app allows teams to monitor and respond to critical system events from anywhere.
+On-call engineers can reimagine incident response with real-time alerts, on-call notifications, and a growing feature set.
+
+## About Grafana OnCall mobile app
+Grafana OnCall for iOS and Android allows you to:
+
+- Override Do Not Disturb on your device to ensure delivery of critical alerts
+- View and troubleshoot alert details from your mobile device
+- Receive push notifications according to your personal notification settings
+- Login with a quick and secure QR code authorization
+
+The OnCall mobile app allows users to receive push notifications as part of their notification policy.
+Push notifications are one of the many notification options in Grafana OnCall.Regardless of your notification preferences, it’s recommended that you configure a diverse set of notification methods.
```

Comment:
```suggestion
Push notifications are one of the many notification options in Grafana OnCall. 
 Consider using multiple notification methods (e.g. mobile app and phone/SMS) to remain available in case the internet or phone network degrades connectivity.
```

Reply:
implemented this in my last commit